config/default/user.role.agency_administrator.yml

```diff
@@ -0,0 +1,43 @@
+uuid: a352e518-129e-47f3-99bb-3b6879a5254e
+langcode: en
+status: true
+dependencies:
+  module:
+    - lightning_core
+third_party_settings:
+  lightning_core:
+    description: ''
+id: agency_administrator
+label: 'Agency Administrator'
+weight: 11
+is_admin: null
+permissions:
+  - 'access content overview'
+  - 'access devel information'
+  - 'access foia personnel overview'
+  - 'add foia personnel entities'
+  - 'administer foia personnel entities'
+  - 'administer taxonomy'
+  - 'administer users'
+  - 'assign roles'
+  - 'create agency_component content'
+  - 'create field_agency'
+  - 'delete agency_component revisions'
+  - 'delete all foia personnel revisions'
+  - 'delete any agency_component content'
+  - 'delete foia personnel entities'
+  - 'delete own agency_component content'
+  - 'delete terms in agency'
+  - 'edit any agency_component content'
+  - 'edit field_agency'
+  - 'edit foia personnel entities'
+  - 'edit own agency_component content'
+  - 'edit terms in agency'
+  - 'revert agency_component revisions'
+  - 'revert all foia personnel revisions'
+  - 'use text format rich_text'
+  - 'view agency_component revisions'
+  - 'view all foia personnel revisions'
+  - 'view published foia personnel entities'
+  - 'view the administration theme'
+  - 'view unpublished foia personnel entities'
```

Comment:
Some permissions that should be granted/removed for the administrator:
- Grant `Edit own value for field field_agency` (they're allowed to edit others' values)
- Grant `View own value for field field_agency` (they're allowed to view others' values)
- Grant `View own unpublished content` (since they have the ability to create components)
- Grant `Access the webform overview page` (since these are the users, like Matt, who may need to create/tweak webforms for components)
- Grant `Create webforms` (see above reason)
- Grant `Delete any webform`
- Grant `Delete own webform`
- Grant `Edit any webform`
- Grant `Edit own webform`
- Grant `View any webform submission`
- Grant `View own webform submission`
- Remove `Agency Component: Delete revisions` (wouldn't want audit trails to be wiped away)
- Remove `Delete all revisions` for FOIA personnel entities (same audit trails reason)

Reply:
I'm not sure the field_agency changes are necessary. It looks like `Agency Administrator`s can already edit and view that value in the `Agency Component` node edit form.